lib/config.go

```diff
@@ -89,7 +89,9 @@ func (this *WormholeConfig) AvailableApps() string {
 
 func (this *WormholeConfig) translatePath(path string) string {
 	for from, to := range this.Mapping {
-		path = strings.Replace(path, from, to, 1)
+		if strings.Index(path, from) == 0 {
+			return filepath.FromSlash(strings.Replace(path, from, to, 1))
+		}
```

Comment:
This is necessary so that the translation works correctly in this case:

```yaml
mapping:
  "/mnt/c": "C:"
  "/": "C:/Users/Timm/AppData/Local/lxss/"
```

Otherwise, **/mnt/c/dir/file.txt** gets translated to **c:/dir/file.txt** first (OK) and then the first slash also gets translated to **C:/Users/Timm/AppData/Local/lxss/"**, so we end up with **C:`C:/Users/Timm/AppData/Local/lxss/`dir/file.txt"** (wrong!)